website/app/services/head-data.js

```diff
@@ -28,4 +31,8 @@ export default class CustomHeadDataService extends HeadDataService {
       : 'assets/logos/share-card.jpg';
     return `${config['ember-meta'].url}/${previewImage}`;
   }
+
+  get url() {
+    return `${config['ember-meta'].url}/${this.router.currentURL}`;
```

Comment:
should figure out how to not do this if we are on the home page (it adds `//` currently)